src/map/npc-parse.cpp

```diff
@@ -153,7 +153,7 @@ bool npc_load_warp(ast::npc::Warp& warp)
     nd->dir = DIR::S;
     nd->flag = 0;
     nd->sit = DamageType::STAND;
-    nd->name = warp.name.data;
+    nd->name = stringish<NpcName>(STRPRINTF("w%c%i"_fmt, 6, nd->bl_id));
```

Comment:
@wushin ideally I would just put `nd->name = NpcName()` or omit the line entirely but tmwa whiles when a npc does not have a name so I guess there's no escape?